editor/configs/crashesLayerLabels.ts

```diff
@@ -0,0 +1,20 @@
+import { LayerProps } from "react-map-gl";
+
+export const crashesLayerLabels: LayerProps = {
+  id: "crash-points-labels",
+  type: "symbol",
+  layout: {
+    // "text-field": ["concat", "CR3 ", ["get", "record_locator"]],
```

Comment:
I experimented with prefixing `CR3` and `Non-CR3` into the labels, but i think the `3` adds visual clutter when trying to read the labels. 

<img width="297" height="349" alt="Image" src="https://github.com/user-attachments/assets/383eb183-8c1e-4be2-8929-c31a53f23b07" />